Bottom: 1aef012868c491d970c4e5b6c1bce85ba64ab235
Top:    e0afc151f8aeca6b54cf44051b49e8664721a2ad
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-10-01 13:45:11 +0100

Refresh of toporouter-c-remove-unused-slo

---

diff --git a/src/toporouter.c b/src/toporouter.c
index b732791..cbab571 100644
--- a/src/toporouter.c
+++ b/src/toporouter.c
@@ -767,22 +767,6 @@ tvertex_wind(toporouter_vertex_t  *a, toporouter_vertex_t  *b, toporouter_vertex
   return point_wind(GTS_POINT(a), GTS_POINT(b), GTS_POINT(c));
 }
 
-int 
-sloppy_point_wind(GtsPoint *a, GtsPoint *b, GtsPoint *c) 
-{
-  gdouble rval, dx1, dx2, dy1, dy2;
-  dx1 = b->x - a->x; dy1 = b->y - a->y;
-  dx2 = c->x - b->x; dy2 = c->y - b->y;
-  rval = (dx1*dy2)-(dy1*dx2);
-  return (rval > 10.) ? 1 : ((rval < -10.) ? -1 : 0);
-}
-
-static inline int
-sloppy_vertex_wind(GtsVertex *a, GtsVertex *b, GtsVertex *c) 
-{
-  return point_wind(GTS_POINT(a), GTS_POINT(b), GTS_POINT(c));
-}
-
 /* moves vertex v d units in the direction of vertex p */
 static void
 coord_move_towards_coord_values (double ax, double ay,
